.storybook/config.js

```diff
@@ -12,5 +15,9 @@ addParameters({
   ],
 });
 
-// automatically import all files ending in *.stories.tsx
-configure(require.context('../src', true, /\.stories\.tsx?$/), module);
+function loadStories() {
+  req.keys().forEach(filename => req(filename));
+}
+
+addDecorator(withGlobalStyles);
+configure(loadStories, module);
```

Comment:
There is an issue in this file with the order of when backgrounds are aded. Backgrounds aren't showing on every story now. I think it is probably just moving the addParameters in this file i think